frontend/dev/jsx/layout/container.jsx

```diff
@@ -1,15 +1,9 @@
-import React from 'react'
+import React, {Component} from 'react'
 
-export default class Container extends React.Component {
-  constructor(props) {
-    super(props)
-    this.state = {
-
-    }
-  }
+export default class Container extends Component {
```

Comment:
@tophboogie oops missed this - so I think if we do it with the destructuring, we can eliminate the unused exports (createClass, Children, etc.) from react with webpack (not sure if this is only webpack 2 or not, but it is supposed to be smart enough to do that) we still need the `import React..` for the jsx transpiling, but if you don't use the `React` reference anywhere, webpack only uses it for the jsx in dev and it is not part of the bundle - this way we only get what we are using